tests/fuzz/targets/compile.rs

```diff
@@ -0,0 +1,72 @@
+#![no_main]
+use comemo::Prehashed;
+
+use typst::diag::FileResult;
+use typst::eval::{Bytes, Datetime, Library, Tracer};
+use typst::font::{Font, FontBook};
+use typst::geom::Color;
+use typst::syntax::{FileId, Source};
+use typst::World;
+
+const FONT: &[u8] = include_bytes!("../../../assets/fonts/LinLibertine_R.ttf");
+
+use libfuzzer_sys::fuzz_target;
```

Comment:
Please put all imports here into one block because they're all non-std and non-crate.

Reply:
Done